adbnx_adapter/adbnx_adapter/imdb_arangoDB_networkx_adapter.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Mar 26 09:51:47 2020
+
+@author: Rajiv Sambasivan
+"""
+
+
+from adbnx_adapter.arangoDB_networkx_adapter import ArangoDB_Networkx_Adapter
+import networkx as nx
+#from arango import ArangoClient
+#from adbnx_adapter.custom_http_client import CustomHTTPClient
+
+
+
+class IMDBArangoDB_Networkx_Adapter(ArangoDB_Networkx_Adapter):
+
```

Comment:
weird whitespace

Reply:
@joerg84 ran autopep8 in the updated push